src/game/common/ini.cpp

```diff
@@ -316,10 +308,15 @@ INISection *INIClass::Find_Section(const char *section) const
 {
     captainslog_assert(section != nullptr);
```

Comment:
Don't need this assert as the function has code to handle a null pointer so its an expected possible input.